app/controllers/clients_controller.rb

```diff
@@ -8,6 +8,24 @@ def mypage
   end
   
   def edit
+    @client = current_client
+  end
+
+  def update
+    @client = current_client
+    if @client.update(client_params)
+      flash[:success] = "編集しました。"
+      redirect_to mypage_clients_path
+    else
+      flash.now[:danger] = "編集に失敗しました。"
+      render :edit
+    end
```

Comment:
インスタンス変数のスコープは可能な限り狭めたいです。`redirect_to` の場合は必要ないはずですので。

```suggestion
    if current_client.update(client_params)
      flash[:success] = "編集しました。"
      redirect_to mypage_clients_path
    else
      flash.now[:danger] = "編集に失敗しました。"
      @client = current_client
      render :edit
    end
```

Reply:
勉強になります！
こちらで対応しました！
https://github.com/nisimotoryoga/pbl_fp_matching/pull/69/commits/f161ba4b9abcb44f05db20ba6e15a9482ee1f72f